package.json

```diff
@@ -37,7 +38,8 @@
     "react-router-dom": "^4.3.1",
     "react-spring": "^7.2.8",
     "redux": "^4.0.1",
-    "redux-thunk": "^2.3.0"
+    "redux-thunk": "^2.3.0",
+    "use-force-update": "^1.0.0"
```

Comment:
:thinking:  hmmm...

Reply:
we need to be careful about dependencies. The more we have, the harder it is to update them. This has 5 commits, 3 stars and wasn't updated in months. It's also about 5 lines of code... I've created an issue so we don't miss it https://github.com/BlueBrain/nexus/issues/366